kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/UploadSourceTask.kt

```diff
@@ -105,7 +90,7 @@ class UploadSourceTask : SourceTask() {
 
         logger.info("Polling new records...")
         val records: List<RecordDTO> = try {
-            uploadClient.pollRecords(PollDTO(1, converters.map { it.sourceType }.toSet())).records
+            uploadClient.pollRecords(PollDTO(1, converters.keys)).records
```

Comment:
Aren't the keys here classnames of the converter, whereas I remember the REST resource, fetches based on source-type name?

Reply:
This was a change in the `createConverter`: it returns a pair sourceType to converter. I've made this more explicit in the current class now.